src/comp_loinc/cli.py

```diff
@@ -67,22 +67,22 @@ def callback(self, /, *,
 
         log_dir = self.output_dir / 'logs'
         log_dir.mkdir(parents=True, exist_ok=True)
-        logging.basicConfig(filename=log_dir / f'log_{time.strftime("%Y%m%d-%H%M%S")}.txt',
-                            encoding='utf-8',
-                            level=logging.getLevelName(log_level.upper())
-                            )
+        logging.basicConfig(
+            filename=log_dir / f'log_{time.strftime("%Y%m%d-%H%M%S")}.txt',
+            encoding='utf-8',
+            level=logging.getLevelName(log_level.upper()))
 
         print(self.loinc_release_path)
         print(self.loinc_trees_path)
         print(self.loinc_version)
         print(self.output_dir)
 
     def parts_list(self):
-        print('parts list')
+        print('parts list')  # todo
 
-    @classmethod
-    def _find_loinc_release_path(cls, loinc_dir: Path, work_dir: Path) -> Path:
-        loinc_release_path = None
+    # todo: abstract these _find_*_path funcs
+    @staticmethod
+    def _find_loinc_release_path(loinc_dir: Path, work_dir: Path) -> Path:
```

Comment:
## `@staticmethod`
Was good to mark these with at least `@classmethod`, but `cls` is not used, so these can be static methods, or just isolated functions.